crates/weaver_resolved_schema/src/signal.rs

```diff
@@ -44,17 +45,15 @@ pub struct MultivariateMetric {
     tags: Option<Tags>,
 }
 
-/// An event signal.
+/// An event specification, used for both Span and Log events.
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub struct Event {
     /// The name of the event.
-    name: String,
+    event_name: String,
```

Comment:
I believe just removing this will break weaver for resolving the existing semconv.  We need to do this as an evolution, not a full breaking change.

I'd recommend deprecating fields you're removing and adding in the new ones.

Reply:
I'm surprised that I changed this, it is in the resolved schema so it would be the consumers and I don't see any reason why this would need to be named `event_name` based on the fact that this is a property of an `Event` 😃 . So I'll revert